src/estimagic/visualization/estimation_table.py

```diff
@@ -136,91 +138,94 @@ def estimation_table(
             the resulting table at the given path.
 
     """
-    if not isinstance(models, (tuple, list)):
-        raise TypeError(f"models must be a list or tuple. Not: {type(models)}")
-    models = [_process_model(model) for model in models]
-    model_names = _get_model_names(models)
-    default_col_names, default_col_groups = _get_default_column_names_and_groups(
-        model_names
-    )
-    column_groups = _customize_col_groups(
-        default_col_groups=default_col_groups, custom_col_groups=custom_col_groups
-    )
-    column_names = _customize_col_names(
-        default_col_names=default_col_names, custom_col_names=custom_col_names
-    )
-    show_col_groups = _update_show_col_groups(show_col_groups, column_groups)
-    stats_options = _set_default_stats_options(stats_options)
-    body, footer = _get_estimation_table_body_and_footer(
-        models,
-        column_names,
-        column_groups,
-        custom_param_names,
-        custom_index_names,
-        significance_levels,
-        stats_options,
-        show_col_names,
-        show_col_groups,
-        show_stars,
-        show_inference,
-        confidence_intervals,
-        number_format,
-        add_trailing_zeros,
-    )
-
-    render_inputs = {
-        "body": body,
-        "footer": footer,
-        "render_options": render_options,
-    }
-    if return_type == "render_inputs":
-        out = render_inputs
-    elif str(return_type).endswith("tex"):
-        out = render_latex(
-            **render_inputs,
-            show_footer=show_footer,
-            append_notes=append_notes,
-            notes_label=notes_label,
-            significance_levels=significance_levels,
-            custom_notes=custom_notes,
-            siunitx_warning=siunitx_warning,
-            show_index_names=show_index_names,
-            show_col_names=show_col_names,
-            escape_special_characters=escape_special_characters,
+    with warnings.catch_warnings():
```

Comment:
> But it would not hurt to ignore these warnings preemptively for the entire estimation table function.

@janosg, is this the implementation you had in mind?

Reply:
I would have renamed the current `estimation_table` to `_estimation_table` and defined a new `estimation_table` that calls `_estimation_table` in this catch warnings context. Otherwise we get a lot of indentation.

Reply:
Or decorate it? https://numpy.org/devdocs/reference/generated/numpy.testing.suppress_warnings.html

Reply:
Very good idea! We should do that!

Reply:
sounds good!

Reply:
Thanks!

Reply:
I implemented it using a decorator in the last [commit](https://github.com/OpenSourceEconomics/estimagic/pull/379/commits/a32f69c2c37855d2a63b000402e44e249b3beffa).

Do you have any more suggestions? Otherwise, we can merge.

Reply:
From my side all good!